src/scss/06_components/paragraphs/_event-list.scss

```diff
@@ -0,0 +1,181 @@
+.component--event-list .event {
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: $spacing-half;
+  padding-top: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    padding-bottom: $spacing;
+    padding-top: $spacing;
+  }
+
+  &:first-child {
+    border-top: 1px solid #e5e5e5;
+  }
+
+  > a {
+    display: block;
+    text-decoration: none;
+
+    &:hover {
+      h3 {
+        text-decoration: underline;
+      }
+    }
+
+    @include breakpoint($breakpoint-l) {
+      display: grid;
+      grid-template-columns: 325px 1fr;
```

Comment:
This and other "random" numbers from the design could be as variables in the beginning of file so it's clear were the pixel values come from